adaptive_shield/README.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+SaaS Apps like Office 365, Slack, Zoom, and Salesforce are vital to a business's day-to-day operations, but can often introduce new security challenges. Adaptive Shield's SaaS Security Posture Management solution (SSPM) features proactive, deep, continuous and automated monitoring and management of all business critical SaaS applications. By integrating with all SaaS apps within minutes, Adaptive Shield enables security teams to gain complete control over their SaaS apps, allowing them to harden their security policies and reduce risks.
+
+With the Adaptive Shield integration, you can track and monitor SaaS posture alerts, such as configuration drifts, integration failures, and security check degradations, as Datadog Events.
+
+## Setup
+
+1. Click **Connect Accounts** to begin authorization of this integration. You will then be redirected to [Adaptive Shield][1].
+2. Click on **Settings**.
+3. Go to **Alerts** -> **"Add New Alerts Channel"** -> Select **Datadog**.
```

Comment:
Can you please elaborate on these comments? I'm not sure what you mean by datadog selected and what should be included in this file regarding the site query

Reply:
In the README in steps 2-3, the user is instructed to manually navigate to `Alerts` and then select Datadog. I'm suggesting that the user should instead be taken directly to the Datadog integration page in Adaptive Shield (i.e. the page that shows up after you click `Add New alert` > `Datadog`).

Additionally, there is a `site` query parameter that is included with the `onboarding_url` when the user clicks "Connect Accounts", which indicates the user's datacenter (e.g. `https://app.datadoghq.eu`). This value should be used as the domain for the authorization request that you make from your onboarding page. Adding this parameter to the authorization request will ensure that the user is able to authorize from their correct Datadog region/site.

Reply:
I changed the `onboarding_url` field in the client and the README.md file accordingly. Regarding the `site` query parameter, the team is working on implementing it.

Reply:
Ok, thanks! Let me know if you have any other questions.

Reply:
I just set up a test account with Adaptive Shield to verify the onboarding url, and I think it would be better to use `https://dashboard.adaptive-shield.com/settings/alerts/add/63230b73c9624b93dadf38d4` as the `onboarding_url`, since that   route is specifically for setting up the datadog integration. Does that sound reasonable on your end?

Reply:
The `onboarding_url` is updated to the mentioned url